src/components/Home/HomeMisionVision/HomeMisionVision.css

```diff
@@ -0,0 +1,114 @@
+.home-mision-vision-container{
+      width: 100%;
+      height: 100%;
+      position: relative;
+      background-color: #191E23;
+}
+
+.home-mision, .home-vision{
+      position: absolute;
+      height: 100%;
+      width: 67%; 
+}
+
+.home-mision {
+      left: 0;
+      top: 0;
+      background-image: url(../../../images/home/mision-vision/mision.jpg);
+      -webkit-clip-path: polygon(0 0, 100% 0%, 50% 100%, 0% 100%);
+      clip-path: polygon(0 0, 100% 0%, 50% 100%, 0% 100%); 
+      overflow: hidden;
+      background-size: cover;
+
+}
+
+.home-vision{
+      right: 0;
+      top: 0;
+      background-image: url(../../../images/home/mision-vision/vision.jpg);
+      -webkit-clip-path: polygon(50% 0, 100% 0%, 100% 100%, 0% 100%);
+      clip-path: polygon(50% 0, 100% 0%, 100% 100%, 0% 100%); 
+      overflow: hidden;
+      background-size: cover;
+
+}
+
+.mision-overlay, .vision-overlay{
+      position: relative;
+      top: 0;
+      bottom: 0;
+      left: 0;
+      right: 0;
+      height: 100%;
+      width: 100%;
+      opacity: 1;
+      transition: .5s ease;
+      color:white;
+}
+
+.mision-overlay {
+      background-color: #000000de;
```

Comment:
please change the mission overlay to 000000c0, and the vision overlay to 005f69c0